Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/Delete.ts

```diff
@@ -48,22 +48,20 @@ export async function deleteDocumentByDocumentUuid(
       }
 
       // Extract from the query result
-      const documentAliasMeadowlarkIds: MeadowlarkId[] = documentAliasIdsResult.rows.map((ref) => ref.alias_meadowlark_id);
+      const documentAliasMeadowlarkIds: MeadowlarkId[] = documentAliasIdsResult.map((ref) => ref.alias_meadowlark_id);
 
       // Find any documents that reference this document, either it's own meadowlarkId or an alias
-      const referenceResult: QueryResult<any> = await client.query(
-        findReferencingMeadowlarkIdsSql(documentAliasMeadowlarkIds),
-      );
+      const referenceResult: MeadowlarkId[] = await findReferencingMeadowlarkIdsSql(client, documentAliasMeadowlarkIds);
 
-      if (referenceResult.rows == null) {
+      if (referenceResult == null) {
```

Comment:
This should be a check for empty array. `findReferencingMeadowlarkIdsSql` never returns `null`